lib/cmd/fh3/appforms/environments/submissions/filter.js

```diff
@@ -1,17 +1,20 @@
 var _ = require('underscore');
+var common = require('../../../../../common');
 
 module.exports = {
   'desc': 'Filter Submissions By Form Or Project Id',
   'examples': [{
     cmd: 'fhc appforms environments submissions filter --environment=<Environment Id> [projectid=<Filter By Project ID>] [formid=<Filter By Form ID>]',
     desc: 'Filter Submissions By Form Or Project Id'
   }],
-  'demand': ['environment'],
+  'demand': ['environment', 'page', 'limit'],
```

Comment:
I think so. The API will fall back to defaults, but it makes sense for the user to at least specify them for fhc.